tooling/helpers/rest_helper.py

```diff
@@ -0,0 +1,42 @@
+import requests
+import time
+
+
+def await_transcription_finish(id: str, api_key: str):
+    is_finished = False
+    # TODO: add time cap
+    while not is_finished:
+        # Sleep for 10
+        # This MAY be a bit long but keeps possible impact on performance as low as possible
+        time.sleep(10)
+        r = requests.get(
+            "http://localhost:8393/transcriptions",
```

Comment:
get the port from env file?

Reply:
We can talk about passing it in as a cli param but I dont want to add env files to a benchmarking tool